tests/rados/test_node_drain_customer_bug.py

```diff
@@ -152,6 +158,12 @@ def run(ceph_cluster, **kw):
                 "The traceback messages are noticed in logs.The error snippets are noticed in the MGR logs"
             )
             return 1
+        cmd_unset_unmanaged = (
```

Comment:
Same as above. let's use the new method that will be created.

Reply:
Used new method.